src/CoreWCF.Http/src/CoreWCF/Channels/HttpTransportBindingElement.cs

```diff
@@ -222,6 +224,27 @@ public override T GetProperty<T>(BindingContext context)
             {
                 return (T)(object)new HttpTransportServiceBuilder();
             }
+
+            if (typeof(T) == typeof(IAuthorizationCapabilities))
+            {
+                var binding = context.BindingParameters.OfType<Binding>().FirstOrDefault();
+                if (binding is HttpBindingBase httpBindingBase)
```

Comment:
OfType is a Linq method. BindingParameters is of type BindingParameterCollection which derives from KeyedByTypeCollection\<object\>. You should use Find<Binding>() instead, which returns default if it can't be found. You avoid intermediate IEnumerable instance being created.

Reply:
done here https://github.com/CoreWCF/CoreWCF/pull/850/commits/abc73112939c4622216588547e548eaddc68d36e